cmd/gpu-feature-discovery/main.go

```diff
@@ -86,6 +86,13 @@ func main() {
 			Value:   "/etc/kubernetes/node-feature-discovery/features.d/gfd",
 			EnvVars: []string{"GFD_OUTPUT_FILE"},
 		},
+		&cli.StringFlag{
```

Comment:
Does this need to be added to the config file as well? I don't recall how we treated the path flags.

Reply:
is added to the config `api/config/v1/flags`